package.json

```diff
@@ -63,6 +63,6 @@
     "protractor": "~5.4.0",
     "ts-node": "~8.5.0",
     "tslint": "~5.20.1",
-    "typescript": "~3.9.7"
+    "typescript": "~4.0.2"
```

Comment:
This looks like a bad idea since typescript 4.0 is supported from Angular 10.1 (you're using Angular 10.0 above).